enhancements/two-nodes-openshift/2no.md

```diff
@@ -0,0 +1,516 @@
+---
+title: 2no
+authors:
+  - "@mshitrit"
+reviewers:
+  - "@rwsu"
+  - "@fabbione"
+  - "@carbonin"
+  - "@thomasjungblut"
+  - "@brandisher"
+  - "@DanielFroehlich"
+  - "@jerpeter1"
+  - "@slintes"
+  - "@beekhof"
+  - "@eranco74"
+  - "@yuqi-zhang"
+  - "@gamado"
+  - "@razo7"
+  - "@frajamomo"
+  - "@clobrano"
+
+approvers:
+  - "@rwsu"
+  - "@fabbione"
+  - "@carbonin"
+  - "@thomasjungblut"
+  - "@brandisher"
+  - "@DanielFroehlich"
+  - "@jerpeter1"
+  - "@slintes"
+  - "@beekhof"
+  - "@eranco74"
+  - "@yuqi-zhang"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@jerpeter1"
```

Comment:
Doesn't appear to be one of the API approvers: https://github.com/openshift/api/blob/master/OWNERS

Reply:
resolved